core/src/main/kotlin/com/justai/jaicf/BotEngine.kt

```diff
@@ -234,8 +234,8 @@ class BotEngine(
                                 state.action.execute(this)
                                 withHook(AfterActionHook(botContext, request, reactions, activator, state))
                             } catch (e: Exception) {
+                                logger.warn("Action exception on state ${dc.currentState}", e)
```

Comment:
Let's use `error` log level for exception logging

Reply:
I thought about it. In many cases `error` level is reserved for something critical. I don't think here we should set `error`, just like when we've not found a state, and user request cannot be processed, we use `logger.warn()`.

Reply:
I think, that skipping user's action is critical enough as it can cause problems with not filled properly context or not executed `go` that can make user stuck in some state, and so on. As soon as not found state will cause only skipping single user's request and will not affect scenario logic.
So I'd better log it under `error` level